client/dist/index.html

```diff
@@ -5,6 +5,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Adventure Time</title>
     <link rel="stylesheet" href="styles.css"></link>
+    // using W3 stylesheet for progressbar
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
```

Comment:
Are we allowed to use external styling libraries? Or is this just a simple style import

Reply:
I am actually not sure. Will double check with SEIRs